changelog/issue-2931.md

```diff
@@ -0,0 +1,11 @@
+audience: deployers
+level: major
+reference: issue 2931
+---
+The `secrets_entities` table has been migrated to a relational table `secrets`. The secrets service now requires an additional environment variable `DB_CRYPTO_KEYS` to be set which is a JSON array where each element in an obejct of the form:
```

Comment:
```suggestion
The `secrets_entities` table has been migrated to a relational table `secrets`. The secrets service now requires an additional environment variable `DB_CRYPTO_KEYS` to be set which is a JSON array where each element in an object of the form:
```
@helfi92 and I were talking about this in #3299 -- my understanding was that we wanted to continue to support AZURE_CRYPTO_KEYS, with the option to start using DB_CRYPTO_KEYS when it's time to rotate a secret.  This changelog suggests that AZURE_CRYPTO_KEYS support is being dropped, and also doesn't mention that if existing keys are to remain readable, then the DB_CRYPTO_KEYS properties must have very specific values.